server.js

```diff
@@ -86,4 +80,53 @@ var server = http.createServer(app).listen(port, function () {
 });
 
 
+
+function configureRedis() {
+  if (process.env.REDISTOGO_URL) {
+    var rtg   = require("url").parse(process.env.REDISTOGO_URL);
+    var redis = require("redis").createClient(rtg.port, rtg.hostname);
+    redis.auth(rtg.auth.split(":")[1]);
+    return redis;
+  } else {
+    var redis = require("redis").createClient();
+    return redis;
+  }
+}
+
+function logRedisKeys(redis) {
+  redis.hkeys('polls', function (err, keys) {
+    keys.forEach(function (key) {
+      redis.hget(
+        'polls', key,
+        function(err, value) {
+          console.log(key + ': ' + value);
+        });
+    });
+  });
+}
+
+function storePoll(poll_id, poll, redis) {
+  redis.hmset(poll_id, poll)
+}
+
+
+// TODO move Poll to it's own file
+
+function Poll(poll_id, adminString, voterString, request) {
```

Comment:
I may break it out into prototype methods, i'm thinking the urls should be extracted for sure